src/altinn-app-frontend/src/common/hooks/useProcess.ts

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { useAppDispatch, useAppSelector } from 'src/common/hooks';
+import { selectAppName, selectAppOwner } from 'src/selectors/language';
+import { IsLoadingActions } from 'src/shared/resources/isLoading/isLoadingSlice';
+import { ProcessActions } from 'src/shared/resources/process/processSlice';
+import { QueueActions } from 'src/shared/resources/queue/queueSlice';
+import { ProcessTaskType } from 'src/types';
+
+export function useProcess() {
```

Comment:
Not quite sure what's the point of separating this out to a hook? This should never be re-used, as it would just cause actions to be dispatched multiple times?

Reply:
Good point.
It would possibly be reused if branching to other types of apps in the start of the app. The main point was only to split most of the code out, so it will be easier to isolate later. And see where stuff actually is used.